src/GraphQL/Language/AST/Fields.cs

```diff
@@ -10,85 +7,6 @@ namespace GraphQL.Language.AST
     /// </summary>
     public class Fields : Dictionary<string, Field>
```

Comment:
Let's remove this class. I see only one usage of ` public void Add(Field field)`. Also anyone can call "native" `Add`/`[]` methods from dictionary ignoring "special" `Add(Field field)` method.

Reply:
Sounds good.  I'll do it in a separate PR.

Reply:
#2327